.changes/next-release/bugfix-9834036d-17d5-4064-86da-8769b80dfa9b.json

```
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Amazon Q /doc: remove retry button and instead propose user New Task and End Session buttons to make it in sync with vscode plugin"
```

I think this changelog could be improved, I don't think there's a need to mention vs code to customers here

```suggestion
  "description" : "Amazon Q /doc: Prompt users to start a new documentation task if current task failed"
```

---

Updated